topology/probes/istio/virtualservice.go

```diff
@@ -48,3 +50,126 @@ func (h *virtualServiceHandler) Dump(obj interface{}) string {
 func newVirtualServiceProbe(client *kiali.IstioClient, g *graph.Graph) k8s.Subprobe {
 	return k8s.NewResourceCache(client.GetIstioNetworkingApi(), &kiali.VirtualService{}, "virtualservices", g, &virtualServiceHandler{})
 }
+
+type virtualServicePodLinker struct {
+	graph    *graph.Graph
+	vsCache  *k8s.ResourceCache
+	podCache *k8s.ResourceCache
+}
+
+func getAppsAndVersionsByVirtualService(vs *kiali.VirtualService) ([]string, []string) {
```

Comment:
What about defining a `struct virtualServiceInfo` with the app and the version

Reply:
DONE